librad/Cargo.toml

```diff
@@ -28,13 +28,13 @@ rcgen = "0.8"
 regex = "1.3"
 serde_bytes = "0.11"
 serde_json = "1.0"
-sodiumoxide = "0.2"
 thiserror = "1.0"
 tracing = "0.1"
 tracing-futures = "0.2"
 unicode-normalization = "0.1"
 urltemplate = "0.1"
 webpki = "0.21"
+ed25519-zebra = { version = "2.1.2", path = "../../ed25519-zebra" }
```

Comment:
`ls -la ../../../../etc/shadow` aah interesting

Reply:
Local first.

Reply:
Now I regret not having used `cat`.

Reply:
:joy: